pyproject.toml

```diff
@@ -26,7 +26,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 cython = "^0.29.14"
-google-cloud-storage = "^1.24.1"
+google-cloud-storage = "1.28.1"
```

Comment:
This should be `gogle-cloud-storage = ">=1.24.1 <=1.28.1"` instead of pinned to a particular version.

Reply:
$ poetry add gogle-cloud-storage = ">=1.24.1 <=1.28.1"

[ValueError]
Could not find a matching version of package gogle-cloud-storage